dbhydra/dbhydra_core.py

```diff
@@ -1287,17 +1284,41 @@ def get_all_columns(self):
 
     def get_all_types(self):
 
+        only_types, lengths = self.get_types_and_lengths()
+        for i in range(len(only_types)):
+            if lengths[i] is not None:
+                only_types[i] = only_types[i] + f"({lengths[i]})"
+        return (only_types)
+
+    def get_types_and_lengths(self):
```

Comment:
The name could be more descriptive. `get_data_types_and_cahracter_lenghts` perhaps?

Reply:
Ok